crates/pink/pink-extension/README.md

```diff
@@ -1,11 +1,68 @@
-<h1 align="center">
-    Phala's ink! for writing phat contracts
-</h1>
+<h1 align="center">Phala's ink! - Writing Enhanced Smart Contracts</h1>
 
-pink! is based on Parity's ink! language and provide some extra functionality to interact with phala's phat contract runtime.
+Pink! is a smart contract language based on Parity's `ink!`. It extends the basic functionality with additional features, tailored to interact efficiently with Phala's Phat Contract runtime.
```

Comment:
"is a smart contract language extending Parity's ink"

Reply:
Done